Content.Client/Chemistry/UI/ChemMasterWindow.xaml.cs

```diff
@@ -24,6 +24,8 @@ namespace Content.Client.Chemistry.UI
     public sealed partial class ChemMasterWindow : FancyWindow
     {
         [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+        [Dependency] private readonly IEntityManager _entityManager = default!;
+
```

Comment:
```suggestion

private readonly SpriteSystem _sprite;
```
naming convention for systems is that the name should not include the "System" part